### PR TITLE
Fix remove abort effect

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

**Comment by itkg-alavieille** [comment]
:+1:  but think to update also FolderConfigurationButtonView and FolderDeleteButtonView

**Comment by itkg-nanne** [comment]
some changes @itkg-alavieille

File: BackofficeBundle/Resources/public/coffee/page/NodeView.coffee [hunk 1]
```diff
@@ -29,19 +29,20 @@ NodeView = OrchestraView.extend(
...
-    $('.js-widget-title', @$el).html @renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/elementTitle',
-      element: @options.node
-    )
-    @addAreasToView(@options.node.get('areas'))
-    @addConfigurationButton()
-    if !@options.editable
-      $('.js-widget-blockpanel', @$el).hide()
-    else
-      @addListBlockToView()
+    if appRouter.extractParameters()[0] == @options.node.attributes.node_id
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/page/NodeView.coffee]: may be check if index 0 exist

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**